src/SFA.DAS.EmployerAccounts.Web.UnitTests/Controllers/EmployerTeamControllerTests/WhenCallToActionToggleIsEnabled/WhenSetupSingleApprenticeStatus.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System.Collections.Generic;
+using System.Web.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Authentication;
+using SFA.DAS.Authorization.Services;
+using SFA.DAS.EAS.Portal.Client;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Models.Reservations;
+using SFA.DAS.EmployerAccounts.Web.Controllers;
+using SFA.DAS.EmployerAccounts.Web.Orchestrators;
+using SFA.DAS.EmployerAccounts.Web.ViewModels;
+
+namespace SFA.DAS.EmployerAccounts.Web.UnitTests.Controllers.EmployerTeamControllerTests.WhenCallToActionToggleIsEnabled
+{
+    public class WhenSetupSingleApprenticeStatus
+    {
+        private EmployerTeamController _controller;
+
+        private Mock<IAuthenticationService> mockAuthenticationService;
+        private Mock<IAuthorizationService> mockAuthorizationService;
+        private Mock<IMultiVariantTestingService> mockMultiVariantTestingService;
+        private Mock<ICookieStorageService<FlashMessageViewModel>> mockCookieStorageService;
+        private Mock<EmployerTeamOrchestrator> mockEmployerTeamOrchestrator;
+        private Mock<IPortalClient> mockPortalClient;
+
+        [SetUp]
+        public void Arrange()
+        {
+            mockAuthenticationService = new Mock<IAuthenticationService>();
+            mockAuthorizationService = new Mock<IAuthorizationService>();
+            mockMultiVariantTestingService = new Mock<IMultiVariantTestingService>();
+            mockCookieStorageService = new Mock<ICookieStorageService<FlashMessageViewModel>>();
+            mockEmployerTeamOrchestrator = new Mock<EmployerTeamOrchestrator>();
+            mockPortalClient = new Mock<IPortalClient>();
+
+            mockAuthorizationService.Setup(m => m.IsAuthorized("EmployerFeature.HomePage")).Returns(false);
+            mockAuthorizationService.Setup(m => m.IsAuthorized("EmployerFeature.CallToAction")).Returns(true);
+
+            _controller = new EmployerTeamController(
+                mockAuthenticationService.Object,
+                mockMultiVariantTestingService.Object,
+                mockCookieStorageService.Object,
+                mockEmployerTeamOrchestrator.Object,
+                mockPortalClient.Object,
+                mockAuthorizationService.Object);
+        }
+
+        [Test]
+        public void ThenForNonLevyTheYourSingleApprovedApprenticeViewIsReturnedAtRow1Panel1()
+        {
+            //Arrange
+            var model = new AccountDashboardViewModel()
+            {
+                PayeSchemeCount = 1,
+                ApprenticeshipEmployerType = Common.Domain.Types.ApprenticeshipEmployerType.NonLevy,
+                CallToActionViewModel = new CallToActionViewModel
+                {
+                    Reservations = new List<Reservation> { new Reservation { Status = ReservationStatus.Completed } },
+                    ApprenticeshipsCount = 1
```

Comment:
What will be the CohortStatus?

Reply:
CohortStatus not required for  the Live Apprentice